hw2/03-count.js

```diff
@@ -1,3 +1,19 @@
 // Add your code here
+const originalText = document.getElementById('text').innerHTML;
 
-input.addEventListener('keydown', handleKeyDown);
+const handleKeyDown = (event) => {
+	document.getElementById('text').innerHTML = originalText;
+	const searchKey = document.getElementById('searchKey').value;
+	console.log('EVENT: ', searchKey);
+	const words = originalText.split(' ');
+	for (let i = 0; i < words.length; i++) {
+		if (words[i] === event.target.value)
+			words[i] = `<span style="background-color: yellow">${searchKey}</span>`;
+	}
+	console.log(words);
+	document.getElementById('text').innerHTML = words.join(' ');
+};
+
+const input = document.querySelector('input');
+
+input.addEventListener('keyup', handleKeyDown);
```

Comment:
AirBnb style guide [(19.1)](https://github.com/airbnb/javascript#whitespace--spaces) recommends using soft tabs (2 spaces) for function indentation.